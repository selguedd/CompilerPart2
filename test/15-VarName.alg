variable 1notvariable var1able var1ab1e