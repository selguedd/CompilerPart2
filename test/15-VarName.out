token: variable	lexical unit: VARNAME
token: 1	lexical unit: NUMBER
token: notvariable	lexical unit: VARNAME
token: var1able	lexical unit: VARNAME
token: var1ab1e	lexical unit: VARNAME
Identifiers
notvariable	1
var1ab1e	1
var1able	1
variable	1
